app/sdk/converter/ObjectConverter.java

```diff
@@ -135,7 +135,7 @@ private static ParserContext getParserContext() {
             attributeMeta = inferMetaData(index, fieldClass);
         }
         if (!isFieldClassSupportedForType(fieldClass, attributeMeta.getAttributeType())) {
-            throw new UnsupportedAttributeException(fieldClass, attributeMeta.getAttributeType());
+//            throw new UnsupportedAttributeException(fieldClass, attributeMeta.getAttributeType());
         }
```

Comment:
I don't think this error should be ignored.

Reply:
I don't think it should be either

Reply:
Whoops, you're right. That was accidentally commited